app/lobby/lobby.tsx

```
@@ -0,0 +1,211 @@
+  }));
+
+  // Animate cards on mount
+  React.useEffect(() => {
+    multiplayerScale.value = withDelay(100, withSpring(1, { damping: 6 }));
+    aiScale.value = withDelay(250, withSpring(1, { damping: 6 }));
+    lessonsScale.value = withDelay(400, withSpring(1, { damping: 6 }));
+  }, []);
+
+  // Floating pieces shared values
+  const floatingPieces = Array.from({ length: 6 }).map((_, i) => ({
+    translateY: useSharedValue(0),
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)